core/src/main/scala/zio/schema/StandardType.scala

```diff
@@ -15,7 +17,7 @@ object StandardType {
   implicit object LongType   extends StandardType[Long]
   implicit object FloatType  extends StandardType[Float]
   implicit object DoubleType extends StandardType[Double]
-  implicit object ByteType   extends StandardType[Byte]
+  implicit object ByteType   extends StandardType[Chunk[Byte]]
```

Comment:
Maybe we should change the name here to `BinaryType` instead of `ByteType`?

Reply:
That makes sense. I'll change this.